app/sidekiq/form1010cg/submission_job.rb

```diff
@@ -15,6 +19,13 @@ class SubmissionJob
 
     sidekiq_retries_exhausted do |msg, _e|
       StatsD.increment("#{STATSD_KEY_PREFIX}failed_no_retries_left", tags: ["claim_id:#{msg['args'][0]}"])
+      StatsD.increment('silent_failure_avoided_no_confirmation', tags: DD_ZSF_TAGS)
+
+      claim = SavedClaim::CaregiversAssistanceClaim.find(msg['args'][0])
+      if claim.parsed_form.dig('veteran',
+                               'email') && Flipper.enabled?(:caregiver_use_va_notify_on_submission_failure)
```

Comment:
🎨 not sure if this is a readability win or not, but would a guardclause to avoid the longer line help here?
```suggestion
      return unless Flipper.enabled?(:caregiver_use_va_notify_on_submission_failure)
      
      if claim.parsed_form.dig('veteran', 'email')
```

Reply:
wait - this isn't even inside a method is it? uh, probably never mind then.

Reply:
Yeah, it looks like there are issues with the return from within the `sidekiq_retries_exhausted` here. I did go ahead and just wrap the claim and send email logic in a conditional based on the toggle. I think that reads better and will also make the diff for when we remove the toggle a little cleaner.

Reply:
oh that reads way better for me. Nice!